core/src/main/scala/com/databricks/labs/remorph/parsers/snowflake/rules/SnowflakeCallMapper.scala

```diff
@@ -62,6 +64,10 @@ class SnowflakeCallMapper extends ir.CallMapper with ir.IRHelpers {
       case ir.CallFunction("POSITION", args) => ir.CallFunction("LOCATE", args)
       case ir.CallFunction("REGEXP_LIKE", args) => ir.RLike(args.head, args(1))
       case ir.CallFunction("REGEXP_SUBSTR", args) => regexpExtract(args)
+      case ir.CallFunction("SEQ1", args) => ir.MonotonicallyIncreasingID()
```

Comment:
I'm not convinced with this. given the output of each `seq` bytes are fixed.

Reply:
I agree, But Databricks doesn't have any logic other than MonotonicallyIncreasingID. I would be happy to modify the logic if we have any proper solutions in Databricks.

Reply:
Let me think, 
fyi.. @vil1  another one to ponder on.

Reply:
`SEQ1(1)` -> `SELECT CAST (monotonically_increasing_id() % 128 AS TINYINT);` perhaps

Reply:
We also need to handle this: If the optional sign argument is 0, the sequence continues at 0 after wrap-around. If the optional sign argument is 1, the sequence continues at the smallest representable number based on the given integer width.

Reply:
Yeah, that will also need to be accounted for (didn't bother to do so in my above comment). It may also be slightly more complicated in terms of types as [this](https://docs.snowflake.com/en/sql-reference/data-types-numeric#int-integer-bigint-smallint-tinyint-byteint) suggests that numeric types like `TINYINT` are more of a "hint" in Snowflake.
But in any case, I'm confident we can come up with a faithful translation of `SEQ1` *et al.* using the proper mix of `MONOTONICALLY_INCREASING_ID` and basic arithmetics. 

What is almost sure though, is that this will likely fail any equality or hash based reconciliation process, as the only guarantee both `SEQ` and `MONOTONICALLY_INCREASING_ID` provide is that the number they produce increases monotonically (after having output `42` either one could correctly output any number greater or equal to `43`).

Reply:
I have removed the SEQ from this PR. I will create a new PR for the same